Exiled.API/Features/Toys/Light.cs

```diff
@@ -80,9 +81,10 @@ public bool ShadowEmission
         /// <param name="scale">The scale of the <see cref="Light"/>.</param>
         /// <param name="spawn">Whether the <see cref="Light"/> should be initially spawned.</param>
         /// <returns>The new <see cref="Light"/>.</returns>
+        [Obsolete("Use Create(Vector3, Vector3, Vector3, bool, Color) instead.", true)]
```

Comment:
You don't need to obsolete this

Reply:
Signature won't match, compiler will ask you which method to use. Test it by yourself.

Reply:
we just don't need to remove this method ?